src/gui/accountmanager.cpp

```diff
@@ -33,7 +33,7 @@ static const char urlC[] = "url";
 static const char authTypeC[] = "authType";
 static const char userC[] = "user";
 static const char httpUserC[] = "http_user";
-static const char davUserC[] = "dav_user";
+static const char davUserC[] = "dav_user_id";
```

Comment:
I am not convinced this is needed because in case the webflow login is different from the dav user id `dav_user` is already empty

Reply:
As discussed in the chat it should be fine if we leave it as `dav_user`.